velodyne_driver/package.xml

```diff
@@ -1,39 +1,47 @@
 <?xml version="1.0"?>
-<package format="2">
-  <name>velodyne_driver</name>
-  <version>1.5.2</version>
-  <description>
-    ROS device driver for Velodyne 3D LIDARs.
-  </description>
-  <maintainer email="jwhitley@autonomoustuff.com">Josh Whitley</maintainer>
-  <maintainer email="brice.rebsamen@gmail.com">Brice Rebsamen</maintainer>
-  <author>Jack O'Quin</author>
-  <author>Patrick Beeson</author>
-  <author>Michael Quinlan</author>
-  <author>Yaxin Liu </author>
-  <license>BSD</license>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>velodyne_driver</name>
+    <version>1.5.2</version>
+    <description>
+        ROS device driver for Velodyne 3D LIDARs.
+    </description>
+    <maintainer email="jwhitley@autonomoustuff.com">Josh Whitley</maintainer>
+    <maintainer email="brice.rebsamen@gmail.com">Brice Rebsamen</maintainer>
+    <author>Jack O'Quin</author>
+    <author>Patrick Beeson</author>
+    <author>Michael Quinlan</author>
+    <author>Yaxin Liu</author>
+    <author>Andreas Klintberg</author>
 
-  <url type="website">http://www.ros.org/wiki/velodyne_driver</url>
-  <url type="repository">https://github.com/ros-drivers/velodyne</url>
-  <url type="bugtracker">https://github.com/ros-drivers/velodyne/issues</url>
+    <license>BSD</license>
+    
+    <url type="website">http://www.ros.org/wiki/velodyne_driver</url>
+    <url type="repository">https://github.com/ros-drivers/velodyne</url>
+    <url type="bugtracker">https://github.com/ros-drivers/velodyne/issues</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
-
-  <build_depend>roslint</build_depend>
-
-  <depend>diagnostic_updater</depend>
-  <depend>dynamic_reconfigure</depend>
-  <depend>libpcap</depend>
-  <depend>nodelet</depend>
-  <depend>pluginlib</depend>
-  <depend>roscpp</depend>
-  <depend>tf</depend>
-  <depend>velodyne_msgs</depend>
+    
+    <buildtool_depend>ament_cmake</buildtool_depend>
+  
+    
+    <depend>rclcpp</depend>
+    <depend>rmw_implementation</depend>
+    <depend>diagnostic_updater</depend>
+    <depend>libpcap</depend>
+    <depend>pluginlib</depend>
+    <depend>console_bridge</depend>
```

Comment:
Change this to `libconsole-bridge-dev` and it will build in CI. See the first line at http://wiki.ros.org/console_bridge for explanation.